src/ADAL.PCL/Flows/AcquireTokenInteractiveHandler.cs

```diff
@@ -247,5 +265,51 @@ protected override bool BrokerInvocationRequired()
 
             return false;
         }
+
+        private string ProcessClaims(string extraQueryParameters, string claims)
```

Comment:
What does this method do? Maybe a brief comment as to what 'processing' is done here.
Also are there no Framework types (or existing library helpers) which might help here with parsing/merging query strings?